### DISCUSSION TITLE
Bring back netstandard2.0 support

### DISCUSSION THREAD

**Discussion by abitzer** [UiPath/corewf#154] [Ideas]
Before the update to .NET 5.0 it was possible to use the library with netstandard2.0.
Could you please bring this feature back?

**Comment by lbargaoanu** [comment]
It's not an overhead we're willing to take on at this time. You can use the last verrsion that works or try to maintain your own fork. Predictions are hard, especially about the future, but .Net standard is history :)

**Reply by PeterBodizsTEKenable** [reply]
Hi @lbargaoanu I would like to migrate my .net framework projects to .netstandard2  then I would continue with latest .net. To achieve this, I would need the version of UiPath.Workflow which supports netstandrad2. Unfortunately, this package is unlisted in nuget.org so I can't see the previous versions.

Is there a list somewhere about the previous versions of this package?

Thanks in advance
